test/e2e/storage/vsphere_volume_perf.go

```diff
@@ -69,25 +66,20 @@ var _ = SIGDescribe("vcp-performance [Feature:vsphere]", func() {
 		namespace = f.Namespace.Name
 
 		// Read the environment variables
-		volumeCountStr := os.Getenv("VCP_PERF_VOLUME_COUNT")
-		Expect(volumeCountStr).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_COUNT is not set")
-		volumeCount, err = strconv.Atoi(volumeCountStr)
+		Expect(VCPPerfVolumeCount).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_COUNT is not set")
+		volumeCount, err = strconv.Atoi(VCPPerfVolumeCount)
 		Expect(err).NotTo(HaveOccurred(), "Error Parsing VCP_PERF_VOLUME_COUNT")
 
-		volumesPerPodStr := os.Getenv("VCP_PERF_VOLUME_PER_POD")
-		Expect(volumesPerPodStr).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_PER_POD is not set")
-		volumesPerPod, err = strconv.Atoi(volumesPerPodStr)
+		Expect(VCPPerfVolumesPerPod).NotTo(BeEmpty(), "ENV VCP_PERF_VOLUME_PER_POD is not set")
+		volumesPerPod, err = strconv.Atoi(VCPPerfVolumesPerPod)
 		Expect(err).NotTo(HaveOccurred(), "Error Parsing VCP_PERF_VOLUME_PER_POD")
 
-		iterationsStr := os.Getenv("VCP_PERF_ITERATIONS")
-		Expect(iterationsStr).NotTo(BeEmpty(), "ENV VCP_PERF_ITERATIONS is not set")
-		iterations, err = strconv.Atoi(iterationsStr)
+		Expect(VCPPerfIterations).NotTo(BeEmpty(), "ENV VCP_PERF_ITERATIONS is not set")
+		iterations, err = strconv.Atoi(VCPPerfIterations)
 		Expect(err).NotTo(HaveOccurred(), "Error Parsing VCP_PERF_ITERATIONS")
 
-		policyName = os.Getenv("VSPHERE_SPBM_GOLD_POLICY")
-		datastoreName = os.Getenv("VSPHERE_DATASTORE")
-		Expect(policyName).NotTo(BeEmpty(), "ENV VSPHERE_SPBM_GOLD_POLICY is not set")
-		Expect(datastoreName).NotTo(BeEmpty(), "ENV VSPHERE_DATASTORE is not set")
+		Expect(SPBMGoldPolicyName).NotTo(BeEmpty(), "ENV VSPHERE_SPBM_GOLD_POLICY is not set")
```

Comment:
Lets change SPBMGoldPolicyName -> SPBMPolicyName

Reply:
We have already have variable SPBMPolicyName

Reply:
Done removed this variable